### PR TITLE
adding unique upload and variable paths to prevent conflicts

### PR DESCRIPTION
**Pull request by thomas-portlandlabs**

### CODE REVIEW

File: appspec.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 0.0
 os: linux
 files:
   - source: /
-    destination: /tmp/codedeployupload
+    destination: /tmp/community/codedeployupload
```
Comment (KorvinSzanto) [inline appspec.yml:5]: Is there no way to do this in a generic way? I don't want to have to maintain separate versions for every site if we don't have to

Comment (thomas-portlandlabs) [reply appspec.yml:5]: I believe you proposed that we generate the appspec.yml file using a template and envsubst. Could I trouble you to provide an example of how we'd do this? Or perhaps approve this PR first so I can make sure my other changes work, then create another PR to switch to the template solution?

File: scripts/before_install.sh [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 then
-  echo "export projectdir=\"/home/forge/community.stage.concretecms.com\"" > "/tmp/.cdvariables";
-  echo "export deploydir=\"/home/forge/community.stage.concretecms.com/releases/$DEPLOYMENT_ID\"" >> "/tmp/.cdvariables";
+  echo "export projectdir=\"/home/forge/community.stage.concretecms.com\"" >> "/tmp/$DEPLOYMENT_ID/.cdvariables";
+  echo "export deploydir=\"/home/forge/community.stage.concretecms.com/releases/$DEPLOYMENT_ID\"" >> "/tmp/$DEPLOYMENT_ID/.cdvariables";
 elif [ "$APPLICATION_NAME" == "community.concretecms.com" ]
 then
-  echo "export projectdir=\"/home/forge/community.concretecms.com\"" > "/tmp/.cdvariables";
-  echo "export deploydir=\"/home/forge/community.concretecms.com/releases/$DEPLOYMENT_ID\"" >> "/tmp/.cdvariables";
+  echo "export projectdir=\"/home/forge/community.concretecms.com\"" >> "/tmp/$DEPLOYMENT_ID/.cdvariables";
```
Comment (aembler) [inline scripts/before_install.sh:17]: In order to facilitate open source development on our community stack, a number of our sites are actually in public repositories, including this one. Are we at all concerned that we're sticking our private directory structure into a file that's in a publicly readable repository?

Comment (aembler) [reply scripts/before_install.sh:17]: Given that Korvin has already approved this approach with another public repository here I'm prepared to overlook this for now: https://github.com/concretecms/documentation.concretecms.org/pull/51/files

**Pull request merged**